lightning/src/ln/channelmanager.rs

```diff
@@ -186,6 +186,8 @@ pub enum PendingHTLCRouting {
 		blinded: Option<BlindedForward>,
 		/// The absolute CLTV of the inbound HTLC
 		incoming_cltv_expiry: u32,
+		/// Path to send the packet to the next hop
+		hops: Vec<RouteHop>
```

Comment:
Does this lock us into using only 1 path for trampoline forwards? Seems like that could impact payment reliability